Mathlib/Algebra/TrivSqZeroExt.lean

```diff
@@ -500,6 +505,16 @@ theorem inr_mul_inl [Semiring R] [AddCommMonoid M] [Module R M] [Module Rᵐᵒ
     show (0 : R) • (0 : M) + op r • m = op r • m by rw [smul_zero, zero_add]
 #align triv_sq_zero_ext.inr_mul_inl TrivSqZeroExt.inr_mul_inl
 
+theorem inl_mul_eq_smul [Semiring R] [AddCommMonoid M] [Module R M] [Module Rᵐᵒᵖ M]
```

Comment:
```suggestion
theorem inr_mul_eq_smul [Semiring R] [AddCommMonoid M] [Module R M] [Module Rᵐᵒᵖ M]
```

Reply:
Huh? The line below is `inl r * x = r • x`, not `inr`.

Reply:
Nevermind, I didn't have my glasses on